src/batch-client/index.ts

```diff
@@ -525,6 +538,21 @@ export class ZimbraBatchClient {
 			}
 		});
 
+	public setRecoveryAccount = ({
+		op,
+		recoveryAccount,
+		recoveryAccountVerificationCode
+	}: SetRecoveryAccountOptions) =>
+		this.jsonRequest({
+			name: 'SetRecoveryAccountRequest',
```

Comment:
Take off the `Request` part of this, it gets appended on to the end automatically